### PR TITLE
API 25.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ext {
-  androidPlugin = 'com.android.tools.build:gradle:2.1.0'
+  androidPlugin = 'com.android.tools.build:gradle:2.2.0'
```
Comment (dlew) [inline dependencies.gradle]: This could be 2.2.2.

Comment (JakeWharton) [inline dependencies.gradle]: True. I only need 2.2.0 or greater for a bug fix. If it goes green I'll update and merge.

File: buildSrc/build.gradle [hunk 2]
```diff
@@ -13,6 +13,7 @@ apply from: rootProject.file('../dependencies.gradle')
 
 repositories {
   mavenCentral()
```
Comment (Shyish) [inline buildSrc/build.gradle]: [As far as I understood](http://stackoverflow.com/questions/24852219/android-buildscript-repositories-jcenter-vs-mavencentral) Jcenter is a superset of mavenCentral, so you should not need this one (same in the above gradle file).

Comment (JakeWharton) [inline buildSrc/build.gradle]: I prefer only mavenCentral. Jcenter is only temporary until Google resumes publishing the plugin there.

**Pull request merged**